bin/init.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 
+# install traefik depdendency
+mkdir temp
+curl --silent -L -o traefik.tar.gz https://github.com/traefik/traefik/releases/download/v2.6.3/traefik_v2.6.3_linux_amd64.tar.gz
+tar -xf traefik.tar.gz -C temp 2>&1 1>/dev/null
```

Comment:
I absolutely love this. Instead of having to keep such a large file in the directory, we just pull it from the github, extract it, and place it where it needs to be. Excellent optimization.